src/main/java/handlers/HandlerGetFileContents.java

```diff
@@ -4,25 +4,24 @@
 
 public class HandlerGetFileContents implements Ihandler {
   private IFileIO fileIO;
-  private ResponseHeadersForContent responseHeaders;
+  private ContentTypeForFileExtension contentType;
```

Comment:
Here is an example of something that I feel probably doesn't have to get injected (referencing back to our conversation from Tuesday?)

This `ContentTypeForFileExtension` is a pretty simple class. Could probably just instantiate it directly.

Just my 2 cents.